### DISCUSSION TITLE
Dual autopilot - autoland possible?

### DISCUSSION THREAD

**Discussion by KaiBVG** [flybywiresim/aircraft#3460] [General]
Last night I watched a video where the guy made an autoland by activating the second autopilot and the FBW-bus did a full autoland with flare-modus armed and fullfilled.
I tried this too today but I'm not able to switch on the second autopilot working together with the first one. They only switched one or the other, but not both at the same time.
Was the feature in the video a future planned one or do I have to do special settings to meet the conditions for autoland? I did the same like the guy but it didn't work.

**Comment by ExampleWasTaken** [comment]
it was a beta

**Reply by KaiBVG** [reply]
Ah, thanks. I t looked very good, hopefully it will be released soon!

**Reply by robinottawa** [reply]
Must have been FSX or some other sim. I think I've seen that too.

**Reply by ExampleWasTaken** [reply]
> 
> 
> Must have been FSX or some other sim. I think I've seen that too.

There is a beta at the moment

**Comment by ChruutvoLuzi** [comment]
Yes it will be coming, patience. There is currently already a WiP Custom Autopilot Branch that is already able to do this. It will eventually come into the installer soon.

**Reply by ChruutvoLuzi** [reply]
If you want to try it already:
https://flybywiresim-packages.nyc3.cdn.digitaloceanspaces.com/vmaster-cfbw-cap/A32NX-master-cfbw-cap.zip

WIP!

**Reply by KaiBVG** [reply]
Thanks a lot; I'll try it tommorrow!
Hopefully the upcoming sim update doesn't knock out it. ;

**Reply by snkeyez95** [reply]
> 
> 
> If you want to try it already:
> https://flybywiresim-packages.nyc3.cdn.digitaloceanspaces.com/vmaster-cfbw-cap/A32NX-master-cfbw-cap.zip
> 
> WIP!

Is there an updated link available?

**Reply by ChruutvoLuzi** [reply]
> > If you want to try it already:
> > https://flybywiresim-packages.nyc3.cdn.digitaloceanspaces.com/vmaster-cfbw-cap/A32NX-master-cfbw-cap.zip
> > WIP!
> 
> Is there an updated link available?

Its always the same link, it gets updated on the cdn so you can redownload from this link again

**Reply by Mike777-200ER** [reply]
The above link no longer works for me.  Is there a better link, or has the dual autopilot/autoland feature been included in the Master or cfbw versions?

Thanks!

**Reply by Paytheon** [reply]
The zip link isn't working. Is there somewhere else we can get the auto land feature? Thank you

**Reply by ChruutvoLuzi** [reply]
> The zip link isn't working. Is there somewhere else we can get the auto land feature? Thank you

Yes the cdn has changed, heres the new one: https://api.flybywiresim.com/api/v1/download?url=https://flybywiresim-packages.b-cdn.net/vmaster-cfbw-cap/A32NX-master-cfbw-cap.zip

If you want to stay up to date, check the autoflight channel on discord and its pinned messages

**Reply by Mike777-200ER** [reply]
Thank you!

Mike

**Comment by organawa** [comment]
Hi guys, is there any doc about the autolanding procedure ( for this custom version ) :) ?

**Reply by Mike777-200ER** [reply]
Hi organawa,

Just make sure you have the ILS tuned and GS & LOC indications are showing up on the PFD, and ND in ILS Mode. Be sure to click on LS next to FS.   Get lined up, then press APP, and turn on BOTH autopilots.  This last step is what's new to the autoland version.   Otherwise it's just like any ILS approach.

**Comment by Annolago** [comment]
When will this be released in the dev version (with Custom FBW)? Been quite some time now..

**Reply by ChruutvoLuzi** [reply]
When it is ready. It will come with all other additions such as the new fpm but it will take some time to get mature

**Reply by Annolago** [reply]
I get that, but having multiple Youtube channels show off something coming "soon" and then it is a month or two away is kind of a let down. Maybe not show things before they really are close to completion.
Don't get me wrong, love the product but expectations can kill many things if they are left unanswered..

**Reply by Mike777-200ER** [reply]
The A32NX folks have already made the autoland/dual autopilot version available.  The link is posted earlier in this thread.  It is in a discussion within this thread.  Be sure to click on "view more" within that discussion.  Note that an older posted link no longer works, but the newer one does.  As always, use at your own risk, but I find that it works quite well -- I am using a version I downloaded on February 22nnd.

**Reply by robinottawa** [reply]
Thanks. As someone on that thread pointed out, though, that link doesn't work. Or is unresponsive. If you know of an update, I'd like to try it.

**Reply by Mike777-200ER** [reply]
If you expand that discussion by clicking on "View more," you will find the newer working link, as I already said.  Here it is:
https://api.flybywiresim.com/api/v1/download?url=https://flybywiresim-packages.b-cdn.net/vmaster-cfbw-cap/A32NX-master-cfbw-cap.zip

**Reply by Annolago** [reply]
Hi Mike
Thx, but rather than getting a single version snapshot in time, why not have it available like the other dev versions? After all we are accepting that it is a development version. Don't get me wrong, but I fail to see the logic of having multiple development versions - some available others not. Now I do know the difference between an alfa version and a test version. If that is the case then fine, but again if they can be shown off in public then please don't keep them hidden away for months.

**Reply by ChruutvoLuzi** [reply]
> Hi Mike
> Thx, but rather than getting a single version snapshot in time, why not have it available like the other dev versions? After all we are accepting that it is a development version. Don't get me wrong, but I fail to see the logic of having multiple development versions - some available others not. Now I do know the difference between an alfa version and a test version. If that is the case then fine, but again if they can be shown off in public then please don't keep them hidden away for months.

Because they are sometimes simply not working after they get rebased. To avoid GitHub being flooded with Issues, they have not been made public officially, however, if you really want to use them, nothing is stopping you to build them yourself which just takes a few minutes

**Reply by Mike777-200ER** [reply]
Hi @ChruutvoLuzi,

Thanks for all your work and that of your team!

I don't understand what you mean by "build them yourself."  I just click on the link earlier in this thread and the current cfbw-cap (autoland) zip file downloads.

Mike

**Reply by ChruutvoLuzi** [reply]
> Hi @ChruutvoLuzi,
> 
> Thanks for all your work and that of your team!
> 
> I don't understand what you mean by "build them yourself." I just click on the link earlier in this thread and the current cfbw-cap (autoland) zip file downloads.
> 
> Mike

yeah the cap is prebuilded but there are many branches that arent such as the new fpm. The comment was more targeted towards Annalago. 

Btw, no need to thank me i am just qa 😆

**Reply by mickmcp** [reply]
> If you expand that discussion by clicking on "View more," you will find the newer working link, as I already said. Here it is:
> https://api.flybywiresim.com/api/v1/download?url=https://flybywiresim-packages.b-cdn.net/vmaster-cfbw-cap/A32NX-master-cfbw-cap.zip

Is that link always the latest version or does it change as new additions are added. I downloaded it last week and it was great but now the auto-throttle is misbehaving along with a few other things.

**Reply by Mike777-200ER** [reply]
Hi mickmcp,

As far as I know, that is the current link.  That version does change often, and is different from the development version in several ways, not only in having the dual autopilot/autoland capability.  Please keep in mind that the A32NX folks posted access to that version only as a special favor to people who requested it.

I don't use the installer, but since the autoland version is even more experimental than the other development versions, I am not surprised that the installer doesn't include it

**Comment by Annolago** [comment]
Mike
Just a follow-up beside building the build our self is there a way to keep getting updates on that build. I shortly believed that the installer recognized the different version, but that turned out to not be the case.

**Reply by ExampleWasTaken** [reply]
That build is not supported in any way. You need to look for a new version yourself.
